chart/kubeapps/values.schema.json

```diff
@@ -142,37 +142,6 @@
           }
         }
       }
-    },
-    "mongodb": {
```

Comment:
I am removing this from the schema since the persistence is disabled by default (and it's not really recommended to turn it on) so it didn't make sense to add the clutter of mongo vs postgres.